src/main/java/mops/businesslogic/DirectoryServiceImpl.java

```diff
@@ -123,6 +125,10 @@ public Directory createFolder(Account account, long parentDirId, String dirName)
         Directory rootDirectory = fetchDirectory(parentDirId);
         long groupFolderCount = directoryRepository.getGroupFolderCount(rootDirectory.getGroupOwner());
         if (groupFolderCount >= MAX_FOLDER_PER_GROUP) {
+            log.error(String.format("The user '%s' tried to create another sub folder for the group with the id %d,"
```

Comment:
no need to use string.format, there are overloads of `info`, `warn`, ... which will do the formatting internally